app/views/dashboard/epic_queues/_epic_queue_records_table.html.haml

```diff
@@ -18,22 +18,22 @@
 -# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR~
 -# TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.~
 
-.bootstrap-table-dropdown-overflow
+.card
   #epic-queues-custom-toolbar
   %table.epic-queue-records-table{ data: { toggle: 'table', search: 'true', 'show-columns' => 'true', 'show-refresh' => 'true', 'show-toggle' => 'true', 'show-export' => 'true', pagination: 'true', side_pagination: 'server', url: dashboard_epic_queue_records_path, striped: 'true', toolbar: '#epic-queues-custom-toolbar' } }
     %thead.primary-header
       %tr
         %th{data: { field: "protocol", align: "left", sortable: 'true', class: 'text-primary' } }
```

Comment:
Can you remove the `text-primary` class from this `th`? It makes it seem like there should be a tooltip or some sort of link but there isn't